freva-data-portal-worker/src/data_portal_worker/backends/posix.py

```diff
@@ -0,0 +1,57 @@
+"""Load data from a posix file system."""
+
+from pathlib import Path
+from typing import Optional, Union
+
+import netCDF4
```

Comment:
That shouldn't matter, because those libs only read metadata, not the actual data.

Also moving the imports isn't really critical because this has to be done only once - when starting up the service. Ideally that only happnes once in a while.